example/escape/main.go

```diff
@@ -0,0 +1,26 @@
+package main
```

Comment:
please add little document for it in `example/escape/README.md`.
we should note this is not an example.

Reply:
deleted this 'example'